src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/UsecaseSelection.tsx

```diff
@@ -43,17 +50,14 @@ const UsecaseSelection = (props: { endTimeArray; setEndTimeArray; SREdata; setSR
 
     props.setSREdata({ ...{ ...props.SREdata }, experiment_name: value });
     sessionStorage.setItem('Experiment Name', value);
-    const response =  await fetch(getListExperimentsURLWithParams(value));
+    const response = await fetch(getListExperimentsURLWithParams(value));
     const data = await response.json();
     const experimentUsecase = data[0].experiment_usecase_type;
-    if(experimentUsecase) 
-    {const usecase = Object.keys(experimentUsecase).find(
-      key => experimentUsecase[key] === true
-    );
-console.log(usecase)
-    setExpUsecaseType(usecase);
+    if (experimentUsecase) {
+      const usecase = Object.keys(experimentUsecase).find((key) => experimentUsecase[key] === true);
```

Comment:
Can you consider a case where both local_monitoring and local_experiment can be true ? 

"experiment_usecase_type": {
      "remote_monitoring": false,
      "local_monitoring": true,
      "local_experiment": false
    },

Reply:
sure, i will add this change in my current pr. will it be ok?

Reply:
I feel it should by default consider as many true values